src/pykeen/trackers/wandb.py

```diff
@@ -28,16 +28,17 @@ class WANDBResultTracker(ResultTracker):
     def __init__(
         self,
         project: str,
-        experiment: Optional[str] = None,
         offline: bool = False,
         **kwargs,
     ):
         """Initialize result tracking via WANDB.
 
         :param project:
             project name your WANDB login has access to.
-        :param experiment:
-            The experiment name to appear on the website. If not given, WANDB will generate a random name.
+        :param offline:
+            whether to run in offline mode, i.e, without syncing with the wandb server.
+        :param kwargs:
+            additional keyword arguments passed to :func:`wandb.init`.
```

Comment:
@cthoyt  the usual uncertainty whether this will link properly in the rendered documentation

Reply:
That looks right!